package/ios/RNCSliderComponentView.mm

```diff
@@ -272,6 +267,21 @@ - (void)loadImageFromImageSource:(ImageSource)source completionBlock:(RNCLoadIma
     }
 }
 
+void updateLimits(RNCSlider *slider, float newLowerLimit, float newUpperLimit) {
```

Comment:
I believe this got skipped when applying changes, as we still fallback to the upper limit 🙃